starters/next-wordpress-starter/lib/WordPressClient.js

```diff
@@ -4,3 +4,11 @@ export const client = new GraphqlClientFactory(process.env.backendUrl).create();
 export const paginationClient = new GraphqlClientFactory(
 	'https://test-decoupled-wp-mock-data.pantheonsite.io/wp/graphql',
 ).create();
+
+export const getAuthCredentials = () => {
```

Comment:
Maybe this is worth a unit test, what do you think @enZane?

Reply:
Done!